modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/App.js

```diff
@@ -35,6 +36,7 @@ export default function App({state}) {
 	const initialState = reducer(state, {type: INIT});
 
 	useAutoExtendSession();
+	useIE11Compatibility();
```

Comment:
This is a placeholder name, we can think about a better name later, I wanted to grab your feedback first :)